draft-ietf-dprive-dnsoquic.md

```diff
@@ -893,6 +893,67 @@ the Extended DNS Error Codes registry {{!RFC8914}}:
        Purpose                Too Early
        Reference              This document  
 
+## DNS over QUIC Error Codes Registry {#iana-error-codes}
+
+IANA \[SHALL add/has added] a registry for "DNS over QUIC Error Codes" on the
+"Domain Name System (DNS) Parameters" web page.
+
+The "DNS over QUIC Error Codes" registry governs a 62-bit space.  This space is
+split into three regions that are governed by different policies.  Permanent
+registrations in this registry are assigned using the Specification Required
+policy ({{!RFC8126}}), except for values between 0x00 and 0x3f (in hexadecimal;
+inclusive), which are assigned using Standards Action or IESG Approval as
+defined in Section 4.9 and 4.10 of {{!RFC8126}}.
+
+Provisional registrations of codepoints are intended to allow for private use
+and experimentation with extensions to DNS over QUIC. Provisional registrations
+only require the inclusion of the codepoint value and contact information. However,
+provisional registrations could be reclaimed and reassigned for another purpose.
+Provisional registrations require Expert Review, as defined in Section 4.5 of {{!RFC8126}}.
+The designated expert or experts are advised that only registrations for an excessive
+proportion of remaining codepoint space or the very first unassigned value (see Section 22.1.2)
+can be rejected. Provisional registrations will include a Date field that indicates
+when the registration was last updated. A request to update the date on any provisional
+registration can be made without review from the designated expert(s).
```

Comment:
You cite Section 22.1.2 here, but I think that is down to copy and paste from QUIC.  Maybe you can just reference QUIC and cut down this text.